### DISCUSSION TITLE
No documentation regarding `replace`?

### DISCUSSION THREAD

**Discussion by user31415192** [uBlockOrigin/uBlock-issues#3209] [Q&A]
### Prerequisites

- [X] I verified that this is not a filter list issue. Report any issues with filter lists or broken website functionality in the [uAssets issue tracker](https://github.com/uBlockOrigin/uAssets/issues).
- [X] This is **NOT** a [_YouTube_](https://www.github.com/uBlockOrigin/uAssets/issues/20586), [_Facebook_](https://www.github.com/uBlockOrigin/uAssets/issues/3367) or [_Twitch_](https://www.github.com/uBlockOrigin/uAssets/issues/5184) report. _These sites MUST be reported by clicking their **respective links**_.
- [X] This is not a support issue or a question. For support, questions, or help, visit [/r/uBlockOrigin](https://www.reddit.com/r/uBlockOrigin/).
- [X] I performed a [cursory search of the issue tracker](https://github.com/uBlockOrigin/uBlock-issues/issues?q=is%3Aissue) to avoid opening a duplicate issue.
- [X] The issue is not present after disabling uBO in the browser.
- [X] I checked the [documentation](https://github.com/gorhill/uBlock/wiki) to understand that the issue I am reporting is not normal behavior.

### I tried to reproduce the issue when...

- [X] uBO is the only extension.
- [X] uBO uses default lists and settings.
- [X] using a new, unmodified browser profile.

### Description

I've read through the wiki, and it seems that you have yet to add documentation regarding the `replace` filter option since it was added in [1.53.0](//github.com/gorhill/uBlock/releases/tag/1.53.0) under the name `urltransform` (later renamed to `uritransform` then `replace`). Why is it still the case to this day?

### A specific URL where the issue occurs.

```yaml
https://github.com/gorhill/uBlock/wiki/Static-filter-syntax
```

### Steps to Reproduce

1. Visit the above link.
2. Scour the page for any information regarding this filter option.

### Expected behavior

There should be information about this filter option.

### Actual behavior

Nope: no such information is found anywhere on the page.

### uBO version

1.57.2 (≥ 1.53.0)

### Browser name and version

–

### Operating System and version

–

**Comment by gorhill** [comment]
> Why is it still the case to this day?

Because we invest our personal time and nobody found the time to do it.

Meanwhile you can use AdGuard's documentation: https://adguard.com/kb/general/ad-filtering/create-own-filters/#replace-modifier

**Comment by u-RraaLL** [comment]
`replace` was added in 1.54. It is not the same thing as `url`/`uritransform` either.

https://github.com/gorhill/uBlock/commit/7c3e060c01

**Comment by krystian3w** [comment]
AdGuard seems use `/old/new/g` without write `g` letter, so few AdGuard filters need manually put `g` to test usage like in AdGuard.

Very optional AdGuard didn't notice as bug that their replace works on multiple matches without using `g`.

Your own documentation may be useful in such circumstances.

**Comment by gorhill** [comment]
Closing this, documentation can be discussed in <https://github.com/uBlockOrigin/uBlock-issues/discussions>.
